src/web/mjs/connectors/jpm1234.mjs

```diff
@@ -0,0 +1,42 @@
+import SinMH from './templates/SinMH.mjs';
+
+export default class jpm1234 extends SinMH {
+    constructor() {
+        super();
+        super.id = 'jpm1234';
+        super.label = 'jpm1234';
+        this.tags = [ 'manga','webtoon','chinese' ];
+        this.url = 'http://jpm1234.com';
+        this.requestOptions.headers.set('x-referer', this.url);
+
+        this.path = '/All/';
+        this.pathMatch = '/All/0/0/0/0/0/lastpost/p/(\\d+)/';
+        this.queryMangasPageCount = '#last_page';
+        this.queryChapters = 'div.chapter-list ul li a';
+        this.config = {
+            throttle: {
+                label: 'Throttle Requests [ms]',
+                description: 'Enter the timespan in [ms] to delay consecuitive HTTP requests.\nThe website may reject to many consecuitive requests.\nChapters with lots of pages may require a longer delay',
+                input: 'numeric',
+                min: 0,
+                max: 5000,
+                value: 1000
+            }
+        };
+
+        this.scriptPages = `
+            new Promise(resolve => resolve(cInfo.fs.map(img => 'http://' + (/\\d$/.test(img) ? pageConfig.host.auto[0].replace('uploads','') : pageConfig.host.auto) + img)));
+        `;
+    }
+
+    async _getMangasFromPage(page) {
+        let request = new Request(this.url + this.pathMatch.replace('(\\d+)', page), this.requestOptions);
+        let data = await this.fetchDOM(request, this.queryMangas , 3);
+        return data.filter(datt => datt.href != "hakuneko:").map(element => {
```

Comment:
should i move .filter(datt => datt.href != "hakuneko:") to SinMH.mjs
it would mean i can not overwrite _getMangasFromPage

Reply:
This check should not be necessary, either fix the query to ignore fake links, or filter the list from super
```javascript
const pages = await super._getPages(chapter);
return pages.filter(page => /* ... */);
```